install.rb

```diff
@@ -260,15 +260,12 @@ def prepare_installation
     $operatingsystem = Facter.value :operatingsystem
   end
 
+  # populates constants used to specify default Windows directories
+  require 'win32/dir' if $operatingsystem == "windows"
```

Comment:
Do we care about the old message?  Or is the new Ruby failure to load `win32/dir` message enough to point us in the right direction?